content/usgs-packages/Application_Intro.Rmd

```diff
@@ -32,14 +32,16 @@ set.seed(1)
 
 ## Lesson Summary
 
-TBD
+This lesson will revisit the packages taught in previous lessons and use some of their functions to create a common data analysis workflow in a modular and reproducible manner. Then, you can learn how a modular approach compares to traditional analyses.
+
+What is modularity and why is it more reproducible? Modularity is the idea that your workflow leverages other tools to do some of the more general tasks. You can think of it as a sort of "plug-and-play" scenario, where you are relying on functionality from another developer to perform pieces of your analysis. This type of workflow reduces some of the maintenance burden because you are pushing some of the responsibility for code accuracy and reliability onto the tools. This also means that you should choose your tools wisely and scrutinize the tools you plan to use in your analyses. 
```

Comment:
i think it's a little of both. the biggest part of what makes it reproducible is the fact that it's scripted at all. and then also, in an ideal world tool maintainers keep their tools up to date with the databases and software dependencies they use, so that your own code will keep running for more months/years after you wrote it, so in that sense modularity -> reproducibility. and then there's a little bit of reproducibility coming from the fact that someone else could look at your code and understand it better if you're outsourcing some complexity to other tools, so they could reproduce or even modify your script more readily. but still, i'd say 80% of reproducibility is complementary to, rather than stemming from, modularity.